composites/AlgoliaSearch/SearchResults.js

```diff
@@ -0,0 +1,198 @@
+
+const messages = defineMessages( {
+	noResultsText: {
+		id: "searchresults.noresultstext",
+		defaultMessage: "No results found.",
+	},
+	foundResultsText: {
+		id: "searchresults.foundresultstext",
+		defaultMessage: "Found {resultsCount} { resultsCount, plural, one {result} other {results} }",
+	},
+} );
+
+/**
+ * The title of the search result item.
+ */
+const SearchResultTitle = styled.p`
+	padding-left: 10px;
+	margin: 0;
+	font-size: 1em;
+	font-weight: normal;
+`;
+
+/**
+ * The link to the search result.
+ */
+const SearchResultLink = styled.a`
+	color: ${ colors.$color_black };
+	padding: 10px;
+	
+	&:hover, &:focus {
+		color: ${ colors.$color_pink_dark };
+	}
+`;
+
+/**
+ * Returns a single search result item.
+ *
+ * @param {Object} props Properties of the SearchResult component.
+ * @returns {ReactElement} The rendered search result.
+ */
+export function SearchResult( props ) {
+	let post = props.post;
+
+	return (
+		<Row {...props}>
+			<SearchResultLink href={ post.permalink } onClick={ props.handler }>
+				<SearchResultTitle>{ post.post_title }</SearchResultTitle>
+			</SearchResultLink>
+		</Row>
+	);
+}
+
+SearchResult.propTypes = {
+	handler: PropTypes.func.isRequired,
+	post: PropTypes.object.isRequired,
+};
+
+/**
+ * Wraps the search results.
+ */
+const SearchResultsWrapper = styled.div`
+	margin-top: 20px;
+	clear: both;
+`;
+
+/**
+ * The no results text paragraph.
+ */
+const NoResults = styled.p`
+	margin-left: 10px;
+`;
+
+/**
+ * Create the ReactElement to render a list of search results.
+ *
+ * @param {object} props The React props.
+ *
+ * @returns {ReactElement} A list of search results.
+ */
+class SearchResults extends React.Component {
+
+	/**
+	 * Constructs the component and sets its initial state.
+	 *
+	 * @param {Object} props The props to use for this component.
+	 *
+	 * @constructor
+	 */
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			results: props.results,
+		};
+	}
+
+	/**
+	 * Handles the situation that there are zero results.
+	 *
+	 * @returns {ReactElement|null} A message stating that no results were found or null if no search term was provided.
+	 */
+	handleZeroResults() {
+		if ( this.props.searchString !== "" ) {
+			return this.renderNoResultsFound();
+		}
+
+		return null;
+	}
+
+	/**
+	 * Renders a no results found text.
+	 *
+	 * @returns {ReactElement} The no results found text.
+	 */
+	renderNoResultsFound() {
+		const noResultsText = this.props.intl.formatMessage( messages.noResultsText );
+
+		a11ySpeak( noResultsText );
+
+		return (
+			<NoResults>
+				{ noResultsText }
+			</NoResults>
+		);
+	}
+
+	/**
+	 * Maps the results to SearchResult components.
+	 *
+	 * @param {Object} results The results returned by Algolia.
+	 *
+	 * @returns {Array} Array containing the mapped search results.
+	 */
+	resultsToSearchItem( results ) {
+		return results.map( ( result, index ) => {
+			return <SearchResult
+				height="32px"
```

Comment:
this causes a `height` HTML attribute to be printed out on a `<li>` element, which is not valid HTML:
`Attribute height not allowed on element li at this point.`

I guess this should be investigated in the Table Row component?